### ISSUE TITLE
Redesign of CLI Interactivity

### ISSUE THREAD

**Issue by ghost** [AnimeThemes/animethemes-batch-encoder#22]
Currently, we have a set of arguments and prompts that walk the user through a particular workflow to generate and execute FFmpeg commands.

This approach is rudimentary and doesn't provide the user with a lot of context outside of the help text and docs.

Ideally, we want this project to be friendly to new users without needing to understand the particulars of the workflow. To accomplish this, we want to transform this script into a CLI app.

In short, rather than the user learning what the different execution modes mean and providing these values before being prompted for input files without explanation, we should walk the user through step interactively.

Some projects relevant to the topic:

https://github.com/nunomaduro/termwind
https://github.com/terkelg/prompts
https://github.com/bczsalba/pytermgui
https://github.com/CITGuru/PyInquirer

Feel free to discuss and refine.
